packages/ui-components/src/components/modal/test/modal.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('Modal (end-to-end)', () => {
+	let page: E2EPage;
+
+	describe('when rendering with default props', () => {
```

Comment:
Please add tests for when the close button is clicked and when is clicked outside the modal container.

Reply:
done ;)